File: ITapply/ITapply.Models/SearchObjects/JobPostingSkillSearchObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ITapply.Models.SearchObjects
{
    public class JobPostingSkillSearchObject : BaseSearchObject
    {
        public int? JobPostingId { get; set; }
        public int? SkillId { get; set; }
    }
} 

File: ITapply/ITapply.UI/itapply_mobile/lib/models/search_objects/job_posting_skill_search_object.dart
import 'package:itapply_desktop/models/search_objects/base_search_object.dart';
import 'package:json_annotation/json_annotation.dart';

part 'job_posting_skill_search_object.g.dart';

@JsonSerializable()
class JobPostingSkillSearchObject extends BaseSearchObject {
  int? JobPostingId;
  int? SkillId;

  JobPostingSkillSearchObject({
    this.JobPostingId,
    this.SkillId,
    super.Page,
    super.PageSize,
    super.IncludeTotalCount,
    super.RetrieveAll,
  });

  @override
  Map<String, dynamic> toJson() => _$JobPostingSkillSearchObjectToJson(this);
}